ukrdict/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-
 from .ukrdict import find_word
 
 __all__ = ["find_word"]
+__version__ = '0.1.0'
```

Comment:
You can retrieve the installed dist version using `pkg_resources`. With this, you now have two places to track for updating the version.

Reply:
Need more read about it, because I don't understand why need `two places to track for updating the version`

Reply:
Every time you release a new version you need to change (re-hardcode) both `pyproject.toml` (1) and `__init__.py` (1) -> 1+1=2

Reply:
```
In [3]: import pkg_resources                                                                   

In [4]: pkg_resources.get_distribution("ukrdict").version                                      
Out[4]: '0.1.0'
```
and `tests/ukrdict_test.py` will be like 

```
import pkg_resources

from ukrdict import __version__


def test_version():
    assert __version__ == pkg_resources.get_distribution("ukrdict").version
```
Right @webknjaz ?

Reply:
What are you testing for now? I just don't like the idea of having to change the same thing on multiple places over and over again.